ipn/ipnstate/ipnstate.go

```diff
@@ -23,12 +23,23 @@ import (
 	"tailscale.com/types/key"
 )
 
+type MyStatus struct {
```

Comment:
I think we can just use PeerStatus and not add this new type.

We could add a new field "IsSelf bool" to PeerStatus and just add the peer to the Peer list.

If you think of it as "tailscale status" showing "the peers of the network" rather than "my peers", then "the peers" includes ourselves.

We can then add a "--self=false" flag to suppress the self record, but default to on.

Reply:
Should we keep it as a separate field to make sorting cleaner?

```go
MyStatus *PeerStatus
Peer map[key.Public]*PeerStatus
```

Reply:
That's fine. But let's name it just `Self *PeerStatus` instead of `MyStatus`.